draft-irtf-cfrg-aegis-aead.md

```diff
@@ -85,6 +85,29 @@ informative:
         org: Graz University of Technology
     date: 2020
 
+  FLLW17:
+    title: "Reforgeability of Authenticated Encryption Schemes"
```

Comment:
Do we want to cite [this](https://link.springer.com/chapter/10.1007/978-3-319-59870-3_2) instead?

Reply:
If not, this looks fine.

Reply:
They've been published at the same time, so I assume they are the same. However, the PDF version is far more readable than the publicly accessible content on Springer.

Reply:
The Publication info says 'Minor revision', but it's not clear to me if that's for the ePrint or Springer version. I also like open access though, and the date is similar enough that we can probably leave it.